projects/igniteui-angular/src/lib/paginator/pager.component.html

```diff
@@ -29,7 +29,7 @@
             paginator.resourceStrings.igx_paginator_pager_text
         }}&nbsp;</span
     >
-    <span>{{ paginator.totalPages }}</span>
+    <span>{{ paginator.totalPagesIndicator }}</span>
```

Comment:
PS: Since this is simple enough, it can easily go without one more getter like `paginator.totalPages || 1` :)